### ISSUE TITLE
[Feature Request]add a capture way : just save text only mode without images.

### ISSUE THREAD

**Issue by leeyaunlong** [zadam/trilium-web-clipper#13]
For some cases, I just want to save selected text to trillum, like ctrl+c , then ctrl+shift+v to paste the text to trilium.

**Comment by zadam** [comment]
Hi, can't you do it exactly like you mention - "ctrl+c , then ctrl+shift+v" - that will paste as plain text.

**Comment by leeyaunlong** [comment]
> Hi, can't you do it exactly like you mention - "ctrl+c , then ctrl+shift+v" - that will paste as plain text.

if I use system clipborad to paste , the url cannot be capture to label pageUrl.

**Comment by zadam** [comment]
So you mean web clipper?

**Comment by leeyaunlong** [comment]
> So you mean web clipper?
yes. in chrome ctrl+c , then ctrl+shift+p in the tilium ,it can not paste the attribute pageUrl.
